yash.md

```diff
@@ -2,7 +2,85 @@
 
-* Access Rights for digital content
+Intended Audience: infrastructure team developers
+
+## Argo
+_The administrative discovery interface for Stanford's Digital Object Registry_
+- ask Andrew, John, JCoyne
+  * Specially chosen staff users (many of them accessioneers)
+
+## infrastructure-integration-test
+_A collection of Capybara integration tests for the SDR_
+- ask any infrastructure dev
+
+## Types of Digital objects
+_We started with more, but we're kinda down to three_
+- ask Andrew
+  * APO
+  * Collection
+  * Item
+
+## cocina-models
+_SDR data model written as syntactically validatable with dry-struct, dry-types and openapi_
+- ask JCoyne or JLitt
+
+## dor-services-app
+_In an effort to isolate our code that directly depends on Fedora3, the DSA was created.  It has grown to be a sort of kitchen sink nexus of a great deal of cross-application / cross-service needs, exposed as API calls._
+
+## Workflows
+
+### Workflow Server
+_a Rails App to track the workflows associated with an object, and the state of each step and an error string for failed steps_
+- ask JCoyne
+
+## Robots
+_what we call our individual accessioning processing steps which are grouped into "workflows" and were engineering before messaging queues were mature._
+- ask anyone and hand waving will occur.  Perhaps Peter?  JCoyne?  Naomi?
+
+### Robot Suites
+#### robot-console
+_Tiny app providing the resque console to view the state of our robot redis queues_
+  * e.g. https://robot-console-prod.stanford.edu/overview
+#### common-accessioning
+_Suite of robots that handle the tasks of accessioning digital objects_
+Workflows implemented include `assemblyWF`, `accessionWF`, `goobiWF`, `disseminationWF`, `releaseWF`
+  * what are "The Robots"? - A system for managing the SDR accessioning pipeline, the robots ingest content into the digital repository.  The Robots coordinate and execute updating the SDR metadata store (currently Fedora data streams), generating technial metadata, handing off to the preservation system a copy of each object version, etc.
+  * The robots inherit their worker functionality from [the lybercore gem](https://github.com/sul-dlss/lyber-core/).
+  * They coordinate using Resque and resque-pool, backed by a Redis instance.
+  * Each type of robot has one or more VMs of its own, but all share the same Redis instance.
+  * Robot activity and errors can be monitored via robot-console.
+  * see also: https://github.com/sul-dlss/common-accessioning/ https://github.com/sul-dlss/gis-robot-suite/ https://github.com/sul-dlss/preservation_robots/ https://github.com/sul-dlss/was_robot_suite
+#### preservation_robots
+_robots to preserve objects being accessioned_
+- ask Naomi
+#### gis-robots
+_robots to carry out special processing required for GIS objects_
+- ask anyone as we are all equally ignorant, more or less
+#### was_robot_suite
+_robots to carry out special processing required for WAS seed and crawl objects_
+- ask Naomi or JLitt
+
+## SURI
```

Comment:
Should this be on https://github.com/sul-dlss/infrastructure-team/blob/gh-pages/sdr/index.md ?